# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1234216392 18000
#      Mon Feb 09 16:53:12 2009 -0500
# Node ID 822cab55ca85fc75e7bf8c83879692154161b75b
# Parent  739141cde75a22b9ac9b5b327ea30c61fd357748
Added tag ss-3-1-52 for changeset 739141cde75a

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -33,8 +33,9 @@ fc55a5e1760b832a672d9de3b53888a3d6793d21
 0b91144f9533b05d14fb5692ed67d8c6f500230f ss-2-9-15
 5608935ff5228a90d34fa80dce3dab2462d20fd3 ss-2-9-16
 77de8319c337b39a477d5433348e4597ad61abe0 ss-2-9-17
 9f02c32eb70d3e6d0c42a2fd5f3b862f4d6c6bcc ss-2-9-18
 89d546610556ee47ec634ab8c3e441e1db248873 ss-2-9-19
 274d9642ac1032fcc4996a3cac80cfe5fd20f8a4 release-3-0-0
 af10baa6391596c3713385d7b6c66c93ee935e87 ss-3-1-50
 a5d1e27ee1f4c3fd66c489e83fdb40f92bf1bf43 ss-3-1-51
+739141cde75a22b9ac9b5b327ea30c61fd357748 ss-3-1-52
